simulatingrisk/hawkdovevar/app.py

```diff
@@ -32,10 +34,94 @@
     }
 )
 
+
+def agents_by_risk(model):
+    """plot total number of agents for each risk attitude"""
+    agent_df = model.datacollector.get_agent_vars_dataframe().reset_index().dropna()
```

Comment:
dropna is pretty aggressive, it'll drop any row with a single na. That's probably what you want but just as a note

Reply:
mesa seems to generate a few rows that are fully `na` in the first rounds before there is any data (possibly due to something I'm doing?); I don't think any of the data I'm generating is ever `na`; I suppose I could restrict this to a specific column like agent id, though.